drivers/hmis/spec/requests/hmis/project_enrollments_performance_spec.rb

```diff
@@ -185,9 +185,10 @@
                                              })
 
         expect do
-          _, result = post_graphql(**adjusted_variables) { query }
-          expect(result.dig('data', 'project', 'enrollments', 'nodes').size).to eq(enrollments.size), result.inspect
-        end.to make_database_queries(count: 10..55) # Query count is high due to optional fields, especially "last contact date". Can maybe optimize further
+          response, result = post_graphql(**adjusted_variables) { query }
+          expect(response.status).to eq(200), result.inspect
+          expect(result.dig('data', 'project', 'enrollments', 'nodes').size).to eq(enrollments.size)
+        end.to make_database_queries(count: 10..35) # Query count is high due to optional fields, especially "last contact date". Can maybe optimize further
```

Comment:
great! maybe we can remove this comment? do you know what exactly it was that reduced this count? I remember fiddling with this for a bit when it was still using the data loader, and being confused by the service loading behavior.

Reply:
Ok, I'll remove the comment. I believe the improvement doesn't come from this PR, but from Theron's removal of the preloader behavior that was resetting associations unnecessarily. (Originally in this closed PR https://github.com/greenriver/hmis-warehouse/pull/5302/files#diff-f6f056c65dd6d1f74650260f273816e2ad19011e401c84238a40a3697795e7bc but then got rolled into the Rails 7.1 work that was merged to release-162.)